documentation/staging/content/userguide/platforms/environments.md

```diff
@@ -82,27 +82,78 @@ For more information, see the [Fusion Middleware Licensing Information User Manu
 
 #### Oracle Linux and WebLogic Server images
 
+{{% notice warning %}}
+Oracle strongly recommends using dated Critical Patch Update (CPU) images from the Oracle Container Registry (OCR), 
```

Comment:
Start the note/warning with, "For production deployments, Oracle recommends..."

Reply:
@mriccell Can you take a look and decide? IIRC the previous guidance was that the note applies to all deployments - but it is just more strongly recommended for production deployments.